JASP-Engine/JASP/R/commonMachineLearningClustering.R

```diff
@@ -107,9 +109,14 @@
     } else if(type == "randomForest"){
       clusterResult <- .randomForestClustering(dataset, options, jaspResults)
     }
-    jaspResults[["clusterResult"]] <- createJaspState(clusterResult)
-    jaspResults[["clusterResult"]]$dependOn(options = c("predictors", "noOfClusters","noOfRandomSets", "noOfIterations", "algorithm", "modelOpt", "seed",
-                                                      "maxClusters", "seedBox", "scaleEqualSD", "m", "distance", "linkage", "eps", "minPts", "noOfTrees", "maxTrees", "optimizationCriterion"))
+  })
+
+  if(isTryError(p))
+   JASP:::.quitAnalysis(gettextf("An error occurred in the analysis: %s", .extractErrorMessage(p))) 
```

Comment:
I don't think this will add much to the user experience if we're being honest ;)
All that changes for users is that it goes from an unexpected error with advice to report it, to an error that gives them less information on what to do. 

Let's not implement this, and instead try to figure out on a bug-by-bug basis if we can find out why the bug occurs. If you can't find the cause for some problems, we'll move them to internal and deal with them when we have more time.

Reply:
Should we not keep it in general? Always better to have a try system than none?

Reply:
We have a try system: `try(analysis)`. That's also why JASP doesn't crash hard if an analysis error is not caught. By doing `try(package) .quitAnalysis(`) the only thing that is accomplished is that the `try(analysis)` does nothing

Reply:
I will remove it